christine-anthony.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="css/christine-anthony.css">
+    <title>CSS Timeline</title>
+    <meta charset="utf-8">
+  </head>
+  <body>
+
+    <h1>Brief History of CSS</h1>
+    <ul>
+      <li id="green"><span class="bold-italic">1994</span>: The first work on style sheets started at CERN.</li>
+      <li><span class="bold-italic">December 1996</span>: CSS was proposed as W3C standard:
+        <blockquote>
+          "CSS1 defines the language in which style sheets are written and a set of properties that designers can use to describe how documents should be presented. The properties in CSS1 cover the most common screen-based presentations."
+        </blockquote>
+      </li>
+      <li>November 1997: The W3C released the first public CSS2 Working Draft.</li>
```

Comment:
We need to apply our `<span class="bold-italic">` element to the rest of the dates in our `<li>`s